src/quo/components/tags/context_tag/schema.cljs

```diff
@@ -1,5 +1,6 @@
 (ns quo.components.tags.context-tag.schema
-  (:require [malli.core :as malli]))
+  (:require [malli.core :as malli]
+            [quo.components.avatars.user-avatar.schema :refer [?profile-picture]]))
```

Comment:
hmm, to me it looks like the profile picture schema is not living in the correct place if it's reused in 2 places but set beside a specific component? should it not be somewhere more generic? 

@ilmotta, @clauxx do we have some structure we want to apply for situations like this?

Reply:
could be added to the common schema registry maybe?

Reply:
I think we need a better structure than just common. 
At the very least 
common/category-1
common/category-2
Etc, otherwise we just end up with a load of random schemas lumped in together

Reply:
we do have 2 categories so far: `schema.common` and `schema.re-frame`. For a separate category, just create a new namespace inside `schema`. Maybe smth like `schema.quo`?

Reply:
Nice! I've created a `schema.quo` namespace, lmk what you think 🙏

Reply:
sounds good @ilmotta , common and utils are two hellish naming conventions in any codebase 😆